### PR TITLE
chore: update zone to 0.5.3

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

File: npm-shrinkwrap.clean.json [hunk 1]
```diff
@@ -3,6 +3,12 @@
     "angular": {
       "version": "1.3.5"
     },
+    "angular-animate": {
+      "version": "1.3.5"
+    },
+    "angular-mocks": {
+      "version": "1.3.5"
+    },
```
Comment (pkozlowski-opensource) [inline npm-shrinkwrap.clean.json]: Hmm, how did we end up adding those with zone version update?

Comment (pkozlowski-opensource) [inline npm-shrinkwrap.clean.json]: Oh, I see, it looks like shrinkwrap wasn't run as part of 0648723. So this change is actually good!

**Comment by pkozlowski-opensource** [comment]
LGTM. I'm going to send it to pre-submit as there are people on gitter reporting that discrepancy between package.json and shinkwrapped version prevents them from doing npm install (I don't see those problems on my end, though...).

On top of this I've got another PR in the queue that needs to update package.json as well.

**Comment by mlaval** [comment]
Presubmit queue failed because of the saucelabs job which faced some timeout issue.
After @mhevery fix, it can't happen anymore. This PR should ne submitted again.

**Pull request closed without merging**